meshPointExample.py

```diff
@@ -52,12 +58,26 @@
                 # trimesh expects XYZ points, not ZYX, so we'll swap these axes
                 xyz_point = [point[2], point[1], point[0]] 
 
+        
                 # check to see whether the point is contained within the current mesh
                 if mesh.contains([xyz_point]):
+                    filename = 'C:/Users/emmak/Desktop/' + mesh_name + '.csv'

```

Comment:
Best not to include hard-coded paths on the main branch here. Probably this can just be just the filename with no path, and the file will implicitly be created in the same directory where this script is run.